File: 백준/Bronze/16504. 종이접기/종이접기.java
import java.io.*;

class Main {
    static int nextInt() throws IOException {
        int c, n = System.in.read() & 15;
        while ((c = System.in.read()) > 32) {
            n = (n << 3) + (n << 1) + (c & 15);
        }
        return n;
    }

    public static void main(String[] args) throws IOException {
        int N = nextInt();
        long sum = 0L;
        for (int i = 0; i < N * N; i++) sum += nextInt();
        System.out.print(sum);
    }
}

File: 백준/Bronze/16504. 종이접기/종이접기.rs
use std::io::{self, Read};

fn main() {
    let mut buffer = String::new();
    io::stdin().read_to_string(&mut buffer).unwrap();
    let mut it = buffer.split_whitespace();

    let mut n = it.next().unwrap().parse().unwrap();
    n = n * n;

    let mut sum: i64 = 0;
    for _ in 0..n {
        sum = sum + it.next().unwrap().parse::<i64>().unwrap();
    }
    print!("{}", sum);
}